src/app/content/reducer.spec.ts

```diff
@@ -141,13 +145,16 @@ describe('content reducer', () => {
         route: content,
       },
     }));
-    expect(newState).toEqual({
-      ...omit('page', state),
-      params: {
-        book: 'foo',
-        page: 'new page',
-      },
-    });
+
+    setTimeout(
```

Comment:
the `setTimeout` is going to make the test pass unconditionally, nothing inside the callback matters.